packages/features/teams/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./components/UpgradeScreen";
+export * from "./components/TeamFeatures";
```

Comment:
we wouldnt need this barrel if TeamFeatures would be in UpgradeScreen and UpgradeScreen would be a generic component like EmptyScreen.

delete this file